install/resources/mybb_theme.xml

```diff
@@ -448,6 +448,14 @@ table {
 	color: #999;
 }
 
+.lastpost_subject {
+	display: inline-block;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	width: 200px;
```

Comment:
You should not fix the size here. Table column has its own width specified.
Better use `display: block;` or a `div`.

Reply:
I have tried both and no success...

Reply:
Perhaps @justinsoltesz could give us some insight?

Reply:
@justinsoltesz @effone - any feedback how to solve this?